modules/opp-architecture-installation.adoc

```diff
@@ -2,14 +2,14 @@
 //
 // * architecture/opp-architecture.adoc
 
-:_module-type: PROCEDURE
+:_module-type: CONCEPT
 [id="opp-architecture-installation_{context}"]
 = Install {product-title} products
 
-To install {product-title}, you must install two products in a specific order. Install {ocp} first. Then, install {rh-rhacm}. It does not matter what order you install the remaining products: {quay}, {rh-storage-essentials-first}, and {acs}. See the following detailed installation information for each product:
+To install {product-title}, you must install {ocp} followed by {rh-rhacm}. The additional products are installed by applying an {rh-rhacm} PolicySet: {quay}, {rh-storage-essentials-first}, and {acs}. 
```

Comment:
I'd either say "The following additional products" or "These additional products"